services/director-v2/src/simcore_service_director_v2/api/routes/computations.py

```diff
@@ -164,10 +165,14 @@ async def _get_project_metadata(
 
     parent_node_id = NodeID(current_project_metadata["node_id"])
     parent_node_idstr = NodeIDStr(f"{parent_node_id}")
-    parent_project_id = await project_repo.get_project_id_from_node(parent_node_id)
-    parent_project = await project_repo.get_project(parent_project_id)
-    assert parent_node_idstr in parent_project.workbench
+    try:
+        parent_project_id = await project_repo.get_project_id_from_node(parent_node_id)
+        parent_project = await project_repo.get_project(parent_project_id)
```

Comment:
it is great, but don't you need to put line 176 inside the try catch?

Reply:
I think it should work the way it is here, but for clarity I guess it would be better to move the `return` statement into the try-except block.